yarn-project/simulator/src/public/side_effect_trace.ts

```diff
@@ -162,11 +162,12 @@ export class PublicSideEffectTrace implements PublicSideEffectTraceInterface {
     this.incrementSideEffectCounter();
   }
 
-  public traceUnencryptedLog(contractAddress: Fr, event: Fr, log: Fr[]) {
+  public traceUnencryptedLog(contractAddress: Fr, log: Fr[]) {
     // TODO(4805): check if some threshold is reached for max logs
     const ulog = new UnencryptedL2Log(
       AztecAddress.fromField(contractAddress),
-      EventSelector.fromField(event),
+      // TODO(esau): Use correct event selector ?
+      EventSelector.fromField(new Fr(0)),
```

Comment:
You can do it in a separate PR, but I guess if event selector is being dropped (ie not enshrined) then it would make sense to remove it from UnencryptedL2Log as well (which is I think what the kernel uses).